addons/account/static/tests/x2many_buttons_tests.js

```diff
@@ -0,0 +1,155 @@
+/** @odoo-module **/
+import {click, editInput, getFixture, patchWithCleanup } from "@web/../tests/helpers/utils";
```

Comment:
You should install and configure a linter for JS to take care of those kind of formating issue
```suggestion
import { click, editInput, getFixture, patchWithCleanup } from "@web/../tests/helpers/utils";
```